tests/e2e/DocumentTest.spec.js

```diff
@@ -39,44 +43,48 @@ module.exports = {
       )
       .click('select[name=access] option[value=public]')
       .assert.value('select', 'public')
-      .waitForElementVisible('iframe', 2000)
+      .waitForElementVisible('iframe', 3000)
       .frame(0)
-       .setValue('.mce-content-body', 'dfdfdfdffdfdfdfdfsfdsfsfsdfsdfdfdfdfdfdfdfdfdfdf')
-       .frame(null)
+      .setValue('.mce-content-body', 'dfdfdfdffdfdfdfdfsfdsfsfsdfsdfdfdfdfdfdfdfdfdfdf')
+      .frame(null)
       .click('button[type=submit]')
-      .pause(2000);
+      .waitForElementVisible('.toast', 3000)
+      .assert.containsText('.toast', 'Document saved');
```

Comment:
Yes.
 I have included a test to check that particular document has been added to the list